app/src/androidTest/java/siriuscyberneticscorporation/teachingaid47plus/StudentClassTest.java

```diff
@@ -0,0 +1,46 @@
+package siriuscyberneticscorporation.teachingaid47plus;
+
+import junit.framework.TestCase;
+
+/**
+ * Created by Johannes on 29.04.2016.
+ */
+public class StudentClassTest extends TestCase {
+
+    public void setUp() throws Exception {
+        super.setUp();
+
+    }
+
+    public void tearDown() throws Exception {
+
+    }
+
```

Comment:
remove setup und teardown since its not used in the example test cases for junit